source/agora/consensus/protocol/Nominator.d

```diff
@@ -952,7 +952,7 @@ extern(D):
     protected Signature signBlock (in BlockHeader header) @safe nothrow
     {
         return header.sign(this.kp.secret,
-            this.enroll_man.getOurPreimage(header.height));
+            this.enroll_man.getPreimage(header.height).hash);
```

Comment:
Is this really equivalent? We are hashing a `PreImageInfo`, right?

Reply:
`getPreimage`: `return PreImageInfo(this.enroll_key, this.cycle[height], height);`
`getOurPreimage`: `return this.cycle[height];`

Since the second argument is `hash`, I believe it is, yes.

Reply:
wait, I confused it with `hashFull` 🤦